subscription/src/main/java/org/killbill/billing/subscription/api/SubscriptionApiBase.java

```diff
@@ -78,7 +77,7 @@ public SubscriptionApiBase(final SubscriptionDao dao, final SubscriptionBaseApiS
     protected SubscriptionBaseBundle getActiveBundleForKey(final String bundleKey, final SubscriptionCatalog catalog, final InternalTenantContext context) throws CatalogApiException {
         final List<SubscriptionBaseBundle> existingBundles = dao.getSubscriptionBundlesForKey(bundleKey, context);
         for (final SubscriptionBaseBundle cur : existingBundles) {
-            final List<DefaultSubscriptionBase> subscriptions = dao.getSubscriptions(cur.getId(), ImmutableList.<SubscriptionBaseEvent>of(), catalog, context);
+            final List<DefaultSubscriptionBase> subscriptions = dao.getSubscriptions(cur.getId(), emptyList(), catalog, context);
```

Comment:
I have seen other places where you don't add the static import and use `Collections.emptyList()` - we should try to be consistent, and I think not doing the static import is more readable

Reply:
Ok, lets use `Collections.emptyList()` and `Collections.sort()`